docs/docs/dev/explore/dpp.md

```diff
@@ -1,64 +1,86 @@
 # Data plane proxy
 
-When Kuma (`kuma-cp`) runs, it waits for the data plane proxies to connect and register themselves.
+A **data plane proxy (DPP)** is the part of Kuma that runs next to each workload that is a member of the mesh.
+A DPP is composed of the following components:
+
+- a `Dataplane` entity which defines the configuration of the data plane proxy.
+- a `kuma-dp` binary that runs on each instance that is part of the mesh. This binary spawns the following subprocesses:
+  - `envoy` which will receive configuration from the control-plane to manage traffic correctly 
+  - `core-dns` which will help resolve Kuma specific DNS entries
+
+::: tip
+Data plane proxies are also often called sidecars.
+:::
+
+Since we have one replica of `kuma-dp` for every replica of every service, the number of DPP running quickly adds up. 
+For example, if we have 6 replicas of a "Redis" service, then we must have one instance of `kuma-dp` running alongside each replica of the service, therefore 6 replicas of `kuma-dp` and 6 `Dataplane` entities as well.
 
 <center>
-<img src="/images/docs/0.4.0/diagram-10.jpg" alt="" style="width: 500px; padding-top: 20px; padding-bottom: 10px;"/>
+<img src="/images/docs/0.4.0/diagram-11.jpg" alt="" style="width: 500px; padding-top: 20px; padding-bottom: 10px;"/>
 </center>
 
-## Dataplane Entity
+## Concepts: Inbounds, Outbounds, Tags and Services
 
-A `Dataplane` entity must be passed to `kuma-dp` when instances attempt to connect to the control plane.
-On Kubernetes, this operation is [fully **automated**](dpp-on-kubernetes.md).
-On Universal, it must be executed [**manually**](dpp-on-universal.md).
+### Inbound
 
-To understand why the `Dataplane` entity is required, we must take a step back. As we have explained already, Kuma follows a sidecar proxy model for the data plane proxies, where we have an instance of a data plane proxy for every instance of our services. Each Service and DP will communicate with each other on the same machine, therefore on `127.0.0.1`.
+Inbounds are defined in the `Dataplane` entity and is a combination of a port and a set of tags.
```

Comment:
Is it correct? 
```suggestion
Inbounds are defined in the `Dataplane` entity and are a combination of a port and a set of tags.
```
Shouldn't outbounds be moved after Inbounds to make the same order as in the title?